man/halt.xml

```diff
@@ -137,12 +136,15 @@
   <refsect1>
     <title>Notes</title>
 
-    <para>These commands are implemented in a way that preserves compatibility with
-    the original SysV commands.
-    <citerefentry><refentrytitle>systemctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>
-    verbs <command>halt</command>, <command>poweroff</command>,
-    <command>reboot</command> provide the same functionality with some additional
-    features.</para>
+    <para>These commands are implemented in a way that preserves basic compatibility with the original SysV
+    commands.  <citerefentry><refentrytitle>systemctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>
+    verbs <command>halt</command>, <command>poweroff</command>, <command>reboot</command> provide the same
+    functionality with some additional features.</para>
+
+    <para>Note that on many SysV systems <command>halt</command> used to be synonymous to
+    <command>poweroff</command>, i.e. both commands would equally result in powering-off of the
+    machine. systemd is more accurate here, and <command>halt</command> results in halting the machine only
+    (leaving power on), and <command>poweroff</command> is required to actually power it off.</para>
```

Comment:
Maybe it's time to remove this distinction? Who would want to not power the machine off these days? I'm trying to say that maybe we shouldn't document this to make it even more set in stone, when the verb is generally fully pointless except on 30-years old broken hardware.

Reply:
I am not sure it's worth tweaking that anymore. If we change something I'd probably just de-emphesize "halt" as a whole, i.e. remove it from --help and man page, stop documenting it. It's a non-sensical command, you are right, but instead of just making it do what the name doesn't suggest I think the better approach would be to just hide and never point people to it. i.e. let's reduce redundancy by clipping the unnecessary twigs.

but anyway, let's do or discuss this in another PR maybe.

Reply:
Yep, deprecating halt sounds like a good idea, and yep, that should be a separate PR.